lms/static/sass/shared/_footer-edx.scss

```diff
@@ -36,6 +39,11 @@ footer#footer-edx-v3 {
       &:last-of-type {
         @include margin-right(0);
       }
+
+      &:hover,
+      &:focus {
+        border: none;
```

Comment:
@frrrances the focus state for anchors in being added via the default states so the underline is shown. I removed the bottom border to prevent double underlines.